.mypy.ini

```diff
@@ -1 +0,0 @@
-[mypy]
```

Comment:
Please keep this file. I'm planning on using it to increase type strictness in the near future.